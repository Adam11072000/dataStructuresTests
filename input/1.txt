init 8186
addImage 9588
setLabelScore 3763 5967 8938 183
resetLabelScore 9588 1188 0
deleteImage 9588
setLabelScore 1169 8194 7377 2931
getHighestScoredLabel 7636 -5
addImage -4
getHighestScoredLabel 9092 -7
deleteImage -6
deleteImage 9678
resetLabelScore 2604 3191 1585
addImage -1
mergeSuperPixels 3541 573 -10
addImage -1
addImage 8334
deleteImage -8
setLabelScore 8334 5755 7992 3234
quit