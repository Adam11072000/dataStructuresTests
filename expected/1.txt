init done.
mergeSuperPixels: INVALID_INPUT
getHighestScoredLabel: INVALID_INPUT
addImage: SUCCESS
quit done.
